binding/ios/Leopard.swift

```diff
@@ -102,17 +103,22 @@ public class Leopard {
             throw LeopardInvalidStateError("Leopard must be initialized before processing")
         }
 
-        if !Leopard.supportedAudioTypes.contains((audioPath as NSString).pathExtension.lowercased()) {
-            throw LeopardInvalidArgumentError("File provided is not supported")
-        }
-
-        if !FileManager().fileExists(atPath: audioPath) {
-            throw LeopardInvalidArgumentError("Could not find the audio file at \(audioPath)")
+        var audioPathArg = audioPath
+        if !FileManager().fileExists(atPath: audioPathArg) {
+            audioPathArg = try getResourcePath(audioPathArg)
         }
 
         var cTranscript: UnsafeMutablePointer<Int8>?
-        let status = pv_leopard_process_file(self.handle, audioPath, &cTranscript)
-        try checkStatus(status, "Leopard process failed")
+        let status = pv_leopard_process_file(self.handle, audioPathArg, &cTranscript)
+        do {
+            try checkStatus(status, "Leopard process failed")
+        } catch let error as LeopardInvalidArgumentError {
+            if !Leopard.supportedAudioTypes.contains((audioPath as NSString).pathExtension.lowercased()) {
+                throw LeopardInvalidArgumentError("File provided is not supported")
```

Comment:
Error message should mention the file name and be specific that it is an audio file format issue